EnglishFilter/sections/specific.txt

```diff
@@ -1618,7 +1618,7 @@ travelermaster.com##.__trm__adspot-title-container
 travelermaster.com##div[class^="__trm__ad-med-rec-"]
 mcmcryptos.xyz##.box-main a[target="_blank"] > img
 ||imgur.com/*.gif$domain=himovies.to|pastemytxt.com
-fmovies.app,moviesjoy.to,sflix.to,himovies.to,vidcloud.pro,vidcloud.msk.ru###overlay-container
+rabbitstream.net,fmovies.app,moviesjoy.to,sflix.to,himovies.to,vidcloud.pro,vidcloud.msk.ru###overlay-container
```

Comment:
How to reproduce it? It's already blocked by `rabbitstream.net,rapid-cloud.ru,streamrapid.ru###overlay-container`.